scripts/remote/remote-download-output.sh

```diff
@@ -40,13 +40,18 @@ ssh -v -p $port -i $keyfile ${user}@${host} -o 'StrictHostKeyChecking no' prefix
 	echo $cmd >> remote-download-cmd.log
 	eval $cmd >> remote-download-stdout.log 2>> remote-download-stderr.log
 
-	# matching_files=( $(find . -maxdepth 1 -name '*.zip') )
-	# unzip ${matching_files[0]} -d $download_folder
-	
-	if [ `uname` == 'Darwin' ]; then
-		unzip -o $download_folder/data.zip -d $download_folder
-	else
-		unzip -o $download_folder/data -d $download_folder
+	# find zip file in download folder
+	matching_files=( $(find $download_folder -maxdepth 1 -name '*.zip') )
+
+	# ensure file exists before unzipping
+	if [ ${matching_files[0]} ] && [ -f ${matching_files[0]} ]; then
+		if [ `uname` == 'Darwin' ]; then
+			unzip -o ${matching_files[0]} -d $download_folder
```

Comment:
Not at the moment. This will unzip the first matching zip file and unzip it since we're also only zipping one file as well, but that can be changed to unzip any files in the directory.